transport/stream.go

```diff
@@ -59,8 +58,8 @@ func (dc *duplexConnAdaptor) CloseWrite() error {
 	return dc.StreamConn.CloseWrite()
 }
 
-// WrapDuplexConn wraps an existing [StreamConn] with new Reader and Writer, but
-// preserving the original [StreamConn.CloseRead] and [StreamConn.CloseWrite].
+// WrapConn wraps an existing [StreamConn] with a new [io.Reader] and [io.Writer], but preserves the original
+// [StreamConn].CloseRead and [StreamConn].CloseWrite.
```

Comment:
Hilary had these as [StreamConn.CloseRead] and [StreamConn.CloseWrite].

Reply:
interface method names should not be wrapped in `[]`